src/Skoruba.MultiTenant.Finbuckle/Extensions/TenantInfoExtensions.cs

```diff
@@ -0,0 +1,101 @@
+﻿using Finbuckle.MultiTenant;
+using System;
+
+namespace Skoruba.MultiTenant.Finbuckle.Extensions
+{
+    public static class TenantInfoExtensions
```

Comment:
I could find no code implementing the logic for the IsActive parameter. Please complete the functionality or remove the parameter.

Reply:
This will be implemented with the tenant manage page